applications/tasks.py

```diff
@@ -8,53 +7,119 @@
+                end_datetime=local_end_of_day(datetime.date.fromisoformat(period["end_date"])),
+                is_reservable=False,
+            )
+            for period in date_periods
+            # Overriding closed date periods are exceptions to the normal opening hours
+            if period["override"] and period["resource_state"] == HaukiResourceState.CLOSED.value
+        ]
+
+    return closed_time_spans
+
+
+def _get_recurring_reservation_details(recurring_reservation: RecurringReservation) -> ReservationDetails:
+    application_section = recurring_reservation.allocated_time_slot.reservation_unit_option.application_section
+    application = application_section.application
+
+    reservee_type = ApplicantTypeChoice(application.applicant_type).get_customer_type_choice()
+    contact_person: Person | None = getattr(application, "contact_person", None)
+    billing_address: Address | None = getattr(application, "billing_address", None)
+
+    reservation_details = ReservationDetails(
+        name=recurring_reservation.name,
+        type=ReservationTypeChoice.SEASONAL,
+        reservee_type=reservee_type,
+        state=ReservationStateChoice.CONFIRMED,
+        user=recurring_reservation.user,
+        handled_at=application.application_round.handled_date,
+        num_persons=application_section.num_persons,
+        buffer_time_before=datetime.timedelta(0),
+        buffer_time_after=datetime.timedelta(0),
+        reservee_first_name=getattr(contact_person, "first_name", ""),
+        reservee_last_name=getattr(contact_person, "last_name", ""),
+        reservee_email=getattr(contact_person, "email", ""),
+        reservee_phone=getattr(contact_person, "phone_number", ""),
+        billing_address_street=getattr(billing_address, "street_address", ""),
+        billing_address_city=getattr(billing_address, "city", ""),
+        billing_address_zip=getattr(billing_address, "post_code", ""),
+        purpose=application_section.purpose,
+        home_city=application.home_city,
+    )
+
+    if reservee_type == CustomerTypeChoice.INDIVIDUAL:
+        reservation_details["description"] = application.additional_information
+        reservation_details["reservee_address_street"] = reservation_details["billing_address_street"]
+        reservation_details["reservee_address_city"] = reservation_details["billing_address_city"]
+        reservation_details["reservee_address_zip"] = reservation_details["billing_address_zip"]
+
+    else:
+        organisation: Organisation | None = getattr(application, "organisation", None)
+        organisation_identifier: str = getattr(organisation, "identifier", "") or ""
+        organisation_address: Address | None = getattr(organisation, "address", None)
+
+        reservation_details["description"] = (
+            translate_for_user(_("Core business"), application.user) + f": {organisation.core_business}"
```

Comment:
This occurred to me just now: This was an issue during a previous seasonal booking round, since it can go over the reservation description char limit. I think we figured that the "Core business: " prefix should just be removed, and the description truncated to match the limit. MIght as well make the change here.